dnf/cli/cli.py

```diff
@@ -222,6 +222,9 @@ def do_transaction(self, display=()):
         if trans:
             msg = self.output.post_transaction_output(trans)
             logger.info(msg)
+            for tsi in trans:
```

Comment:
or it can be used:
if any(map(lambda x: x.op_type == dnf.transaction.FAIL  , trans)):
    raise ...